study.md

```diff
@@ -21,18 +21,53 @@ There is no requirement to follow any links from these pages.
 After doing all the required readings please tell us anything you were confused about.
 
 ```md
-<!-- your answer here -->
+Is this correct?: A method is a function that is a property of an object.  Methods can be written as either:
+let obj = {
+  age: function(param1) {
+    code statements;
+  };
+};
+
+or
+
+obj.age(param1) {
+  code statements;
+};
+
+Do both of the above refer to the same method?  Are there any other ways to write a method?
+
+
+
+
 ```
 
 Are there any nagging questions relating to functions that you have?
 
 ```md
-<!-- your answer here -->
+let myFunction = function(param1) {
+  code;
+};
+Is this what a function literal is? Because it has an assignment operator, does that make it a function literal instead of a function (such as obj.function() {} )?
+
+
+
+Is this how callback functions work?
+
+function myFunction() {} // a function is declared
+function callbackFunction(param1, callback) {} // a callback function is declared.  Do you actually write "callback" as the parameter each time you use a callback function?
+callbackFunction(param1, myFunction) {} // when the callback function is called, does the name of the  function to be called back replace the parameter "callback" in callbackFunction?
+
```

Comment:
Here's a code snippet that I wrote to demonstrate callback functions.  Let me know if it's an unclear.
```javascript
// This function has two parameters: greeter (a callback function) and name (a
// string).  It will call "greeter" with "name" as its argument.
function greet(greeter, name) {
  console.log(`Executing greeter callback with ${name} as the argument.`);

  greeter(name);

  console.log('Greeting finished.');
}

// A named function to be used as a callback function.
function hello(name) {
  console.log(`Hello, ${name}.`);
}

// Another named function to be used as a callback function.
function hi(name) {
  console.log(`Hi, ${name}.`);
}

// Calling greet() with hello() as its callback function.
greet(hello, 'Jackson');
// Output:
// Executing greeter callback with Jackson as the argument.
// Hello, Jackson.
// Greeting finished.

// Calling greet() with hello() as its callback function.
greet(hi, 'Jackson');
// Output:
// Executing greeter callback with Jackson as the argument.
// Hi, Jackson.
// Greeting finished.

// Calling greet() with an anonymous function as its callback function.
greet(someName => {
  console.log(`Greetings, ${someName}.`);
}, 'Jackson');
// Output:
// Executing greeter callback with Jackson as the argument.
// Greetings, Jackson.
// Greeting finished.
```